lib/identity_cache/query_api.rb

```diff
@@ -227,7 +227,7 @@ def resolve_cache_miss(id)
         record = self.includes(cache_fetch_includes).reorder(nil).where(primary_key => id).first
```

Comment:
If any of these embedded associations that are included through `cache_fetch_includes` are read from a read slave, then they will be cached with stale data if `self.should_use_cache?` is true.  This could be a problem if a model that uses one database has a cache_has_many with `embed: true` for an association where the associated model uses another database.